lib/pdc/version.rb

```diff
@@ -1,3 +1,3 @@
 module PDC
-  VERSION = '0.1.6'.freeze
+  VERSION = '0.1.7'.freeze
```

Comment:
Sorry for the confusion. As @sthaha said, I meant in a separate commit about 'bump version' in release process.